neptune/internal/storage/storage_utils.py

```diff
@@ -159,6 +198,11 @@ def upload_to_storage(upload_entries, upload_api_fun, upload_tar_api_fun, **kwar
         if uploading_multiple_entries or creating_a_single_empty_dir:
             data = compress_to_tar_gz_in_memory(upload_entries=package.items)
             upload_tar_api_fun(**dict(kwargs, data=data))
+            if progress_indicator is not None:
```

Comment:
Maybe instead of those ifs we can deliver a noop implementation of progress indicator?